app-config-electron/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@app-config/electron",
+  "description": "Exposes app-config values to Electron render processes",
+  "version": "2.6.0",
+  "license": "MPL-2.0",
+  "author": {
+    "name": "Launchcode",
+    "email": "admin@lc.dev",
+    "url": "https://lc.dev"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/launchcodedev/app-config.git"
+  },
+  "main": "dist/index.js",
+  "module": "dist/es/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist",
+    "!**/*.tsbuildinfo",
+    "!**/*.test.*"
+  ],
+  "scripts": {
+    "build": "tsc -b",
+    "build:es": "tsc -b tsconfig.es.json",
+    "clean": "rm -rf dist *.tsbuildinfo",
+    "lint": "eslint src",
+    "fix": "eslint --fix src",
+    "test": "jest",
+    "prepublishOnly": "yarn clean && yarn build && yarn build:es"
+  },
+  "dependencies": {
+    "@app-config/main": "^2.6.0",
+    "@app-config/logging": "^2.6.0",
+    "electron": "13"
```

Comment:
I wonder how far back we could support electron? The APIs we're using aren't very new, right?

Reply:
Pretty sure we could go back to at least 8

Reply:
Does electron even need to be a dependency? If so, should it be a peerDependency?

Reply:
Yeah I think a peerDependency makes sense, we should be using the same version as the actual electron app using this.

It works with electron versions back to 10 which I think is enough